src/main/java/com/marklogic/client/helper/DatabaseClientConfig.java

```diff
@@ -4,7 +4,11 @@
 
 import com.marklogic.client.DatabaseClientFactory.Authentication;
 import com.marklogic.client.DatabaseClientFactory.SSLHostnameVerifier;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
 
+@Component
```

Comment:
I'm ok with that.  My thinking was influenced by that web post I referenced in issue #34.  I think that this is a component that would be used on any MarkLogic application Spring or non-Spring.